packages/client/src/Conversation.test.tsx

```diff
@@ -43,19 +43,15 @@ it("collapses read messages, with the exception of the last message in the conve
   )
   await updates(app)
 
-  expect(
-    app
-      .find("Presentable")
-      .filterWhere(n => n.props().presentable.id === "11")
-      .find(Collapse)
-  ).toHaveProp("in", false)
+  expect(app.find({ presentable: { id: "11" } }).find(Collapse)).toHaveProp(
```

Comment:
I noticed that there were type errors in this test file. I did some research and found this pattern for finding elements by prop values, which is a little more concise.